src/app/server.ts

```diff
@@ -144,32 +140,19 @@ if (process.env.NODE_ENV === 'production') {
         }
     });
 
-    app.get('*', async (req: Request, res: Response) => {
-        // Eg. http://localhost:9000/commentisfree/...
-        try {
-            const url = buildUrlFromPath(req);
-            const { html, ...config } = await fetch(url).then((article) =>
-                article.json(),
-            );
-
-            req.body = config;
-            return renderArticle(req, res);
-        } catch (error) {
-            // eslint-disable-next-line no-console
-            console.error(error);
-        }
-    });
-
     // express requires all 4 args here:
-    app.use((err: any, req: any, res: any) => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    app.use((err: any, req: any, res: any, next: any) => {
         res.status(500).send(`<pre>${err.stack}</pre>`);
     });
 
-    setInterval(() => {
-        recordBaselineCloudWatchMetrics();
-    }, 10 * 1000);
+    if (process.env.DISABLE_LOGGING_AND_METRICS !== "true") {
+        setInterval(() => {
+            recordBaselineCloudWatchMetrics();
+        }, 10 * 1000);
+    }
 
     app.listen(port);
     // eslint-disable-next-line no-console
-    console.log(`Started production server on port ${port}`);
+    console.log(`Started production server on port ${port}\nready`);
```

Comment:
Did you mean to keep the ready here?

Reply:
yeah, lighthouse ci is waiting for that to show up in stdout to know when to kick off the tests!